### PR TITLE
Add blog post for deliverable 73

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Communicate the deliverable to the customers](https://trello.com/c/5ikAwGSZ/1222-communicate-the-deliverable-to-the-customers)

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 342d0a09fbaeab324c2582c314c6a8cf3fc018eb

https://deploy-preview-247--openbuildservice.netlify.com

File: _posts/development/2020-03-17-TODO.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Menus, Menus Everywhere!
+
+In the previous month, we have introduced the menu for various links such as `Profile` or `Your Home Project`.
+This menu got its own fancy name and it is `Places`.
+We are taking it further now by going through pages and grouping up actions like `Create Project` or `Branch Package` into another menu.
+Actions are specific to a page and instead of having them all over the page, we conveniently organize them together.
+This new menu got a quite original name... it's the `Actions` menu.
+Let's be honest, we have outdone ourselves once more!
```
Comment (hennevogel) [inline _posts/development/2020-03-17-TODO.md]: https://internetsarcasm.com/

File: _posts/development/2020-03-24-responsive.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Today the "My tasks" are tables of requests where somehow the user is involved to.
+
+In the future the "My tasks" are going to be more like a *TODO* list where different kind of notifications are going to be displayed. Notifications like comment mentions, review being added, or request being revoked are going to give the user a better overview about what must be done! Stay tuned!
+
+### New design
+
+This is how the _Tasks_ page looked before
+
+<img src="/images/posts/responsive_ux/old_tasks.png" alt="Old version of Tasks page"/>
```
Comment (danidoni) [inline _posts/development/2020-03-24-responsive.md]: Those two screenshots are the other way around. This screenshot is the after

File: _posts/development/2020-03-24-responsive.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+and this is how it looks now
+
+<img src="/images/posts/responsive_ux/notifications_inbox.png" alt="New version of Tasks page"/>
```
Comment (danidoni) [inline _posts/development/2020-03-24-responsive.md]: And this is the before

File: _posts/development/2020-03-24-responsive.md (hunk 2 above)
Comment (danidoni) [reply _posts/development/2020-03-24-responsive.md]: Fixed

File: _posts/development/2020-03-24-responsive.md (hunk 3 above)
Comment (danidoni) [reply _posts/development/2020-03-24-responsive.md]: Fixed

File: _posts/development/2020-03-24-responsive.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+which are displayed at the bottom of the page for small screens and at the top for larger ones.
+
+In the last few weeks we have gone a bit further and decided to add texts under the icons,
+since we considered that only icons were not meaningful enough by their own.
+As you can see in the following screenshots, all the navigation elements are now clear and consistent
+always including icons and texts.
+
+On small screens:
+
+<img src="/images/posts/responsive_ux/small_icon_text_nav.png" alt="Icons and texts on navigation for small screens" width="200" />
```
Comment (saraycp) [inline _posts/development/2020-03-24-responsive.md]: Too short maybe. I'm going to align the sizes...

File: _posts/development/2020-03-24-responsive.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Working towards Responsiveness
```
Comment (hennevogel) [inline _posts/development/2020-03-24-responsive.md]: *yawn* :wink: how about...

- A notification about notifications!
- Fresh Notifications anyone?
- Notification Renovation

**Pull request merged**